trustedApplications/__snapshots__/service.test.ts.snap

```diff
@@ -43,7 +43,7 @@ Array [
   Statement {
     "object": NamedNode {
       "termType": "NamedNode",
-      "value": "http://www.w3.org/ns/auth/acl#Read",
+      "value": "http://www.w3.org/ns/auth/acl#read",
```

Comment:
That's exactly the kind of nitpicking I need, otherwise I'd never learn. Is a "class" an RDF concept, and if so, how do I know when something is a class?

(This is also probably wrong at many other places.)